Source/SourceKittenFramework/SourceLocation.swift

```diff
@@ -13,14 +13,31 @@ import Clang_C
 #endif
 import Foundation
 
-public struct SourceLocation {
+public struct SourceLocation: Comparable {
     public let file: String
     public let line: UInt32
     public let column: UInt32
     public let offset: UInt32
 
-    public func range(toEnd end: SourceLocation) -> NSRange {
-        return NSRange(location: Int(offset), length: Int(end.offset - offset))
+    public func range(toEnd end: SourceLocation) -> ByteRange {
+        return ByteRange(location: ByteCount(Int(offset)), length: ByteCount(Int(end.offset - offset)))
```

Comment:
ByteCount(Int(offset)) -> ByteCount(offset)

ByteCount can be created with Int64

Reply:
offset in this case is an `Int32` and I didn't think it was worth adding a new initializer for that.